_posts/2016-02-10-installing-server-environment-with-powershell.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: Installing server environment with PowerShell
+author: Rinorragi
+excerpt: Automize your IIS webserver installation with PowerShell 
+categories: 
+- EPiServer
+tags: 
+- EPiServer 
+- DOTNET 
+- PowerShell
+---
+Most likely as a reader you are mostly interested about my scripts. Here are our scripts for setting up Windows Server 2012 R2 ready for WebDeployments and running EPiServer ASP.NET site!
+
+* [Example script using modules below](https://github.com/solita/powershell-webdevelopertools/blob/master/scripts/solita_example_server_install.ps1)
+* [Config used by example script](https://github.com/solita/powershell-webdevelopertools/blob/master/scripts/solita_example_server_install_config.xml)
+* [IIS tools module](https://github.com/solita/powershell-webdevelopertools/blob/master/solita-iistools.psm1)
+* [Server tools module](https://github.com/solita/powershell-webdevelopertools/blob/master/solita-servertools.psm1)
+
+## Why automate the installation?
+
+This blog is about how to make your brand new Windows Server ready for WebDeployments with just pressing enter once. Cloud services can make your infrastructure lifecycle handling very easy; still people are often encountering situations where we need to host our ASP.NET applications in virtual machines or directly on physical hardware. On those situations installation procedures in Windows operating systems are often done with some "clickety click" magic that won't take too long. Still the "clickety click" installations have lots of long-term problems: 
+
+* Installations can't be reproduced 
+* Only the one who installed knows how he did it
```

Comment:
Only the installer knows how it was done? tyyliasia, josta voi olla eri mieltäkin